app/views/teams/memberships/index.html.haml

```diff
@@ -6,7 +6,7 @@
       .section
         %h2 Pending Invitations
         %p These are people who have been invited to this team.
-        %table
+        %table.teams-table
```

Comment:
Are you extracting this because you're going to add the component for memberships below?

Reply:
Yes. We have tables in two pages. I've extracted them because I'd want them to look the same.

Reply:
Ah - I'd missed they were in two different pages. Sorry.